src/test/java/be/seeseemelk/mockbukkit/entity/LivingEntityMockTest.java

```diff
@@ -0,0 +1,45 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class LivingEntityMockTest
+{
+
+	private ServerMock server;
```

Comment:
This variable is only ever used within the `setup()` method. There is no need to declare it as a class member.

Reply:
Thats how we always did it, even if it could be converted to a local variable. Should I change this or keep it like this to keep it in line with the other Tests?

Reply:
If its not needed, then dont declare it as a class member.